packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -38,20 +38,17 @@ const noopSelection = () => {
   return false;
 };
 
-const defaultExpandedDefault = [];
```

Comment:
Unintentionally created a singleton here. Spotted it when tests were failing.

Reply:
Are you mutating these? This should definitely work (to prevent creating new references and allocating new objects on every render). Otherwise you're mutating variables passed from an unsuspecting parent.

Reply:
+1 for keeping the variable at the top scope. It seems that it will warn in https://github.com/mui-org/material-ui/blob/8c636faeac62a6fb5811719f5c3aa7b84876158e/packages/material-ui/src/utils/useControlled.js#L31 if we don't.

Reply:
My bad, we use `JSON.stringify(defaultProp)`, so no warning is thrown.